code/modules/surgery/organs/dwarf_organ.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/organ/alcoholvessel //essentially the opposite of the xeno's plasmavessel, but with alcohol
```

Comment:
How about we add livers, add drunkenness processing to livers, and make the dwarf organ be a subtype of liver?